picard/cluster.py

```diff
@@ -499,7 +522,15 @@ def cluster(self, threshold):
                     c = similarity(token_x, token_y)
                     if c >= threshold:
                         heappush(heap, ((1.0 - c), [x, y]))
-            QtCore.QCoreApplication.processEvents()
+            if tagger and (y % status_update_steps) == 0:
+                tagger.window.set_statusbar_message(
+                    N_("Clustering - step %d/3: clustering by %s (%d/%d)" % (self.cluster_type.value,
+                                                                             self.cluster_type_label(),
+                                                                             5*y/status_update_steps,
+                                                                             100)),
+                    {},
+                    timeout=0.01
+                )
```

Comment:
This isn't used properly. Not sure I get it correctly but it should be something like:

```python
status_msg = N_("Clustering - step %(step)d/3: %(cluster_type)s (%(update)d%%)")
mparms = {
    'step': self.cluster_type.value,
    'cluster_type': _(self.cluster_type_label()),
    'update': 5*y/status_update_steps,
}
tagger.window.set_statusbar_message(status_msg, mparms)
```

Explanation: by default, `set_statusbar_message` will call `_()` on passed message format, `mparms` is a dict used to fill named fields (better to use those for translators).
`100` is hardcoded, and doesn't need any translation, since we have `/100` I guess it's a percentage, hence `%%` which is the escaped form of `%`.
I removed 'clustering by' because it would complexify translation (you need to translate full strings, you can't just concatenate, because not all languages work this way), so this simpler form avoids this (but if you insist, I can explain further).

Reply:
Yup, it was supposed to be a percentage but I didn't manage to print it. 
'Clustering by' removal is fine, it was duplicated with the first word anyway. 
I've tried your suggestion, but replaced lowercased type labels with the already existing 'Artist' and 'Album'.